packages/core/src/components/table/table-header-cell/readme.md

```diff
@@ -20,7 +20,7 @@
 
 | Event     | Description                                                                                                                                                          | Type                                                                                      |
 | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| `tdsSort` | Sends unique Table identifier, column key and sorting direction to the tds-table-body component, can also be listened to in order to implement custom-sorting logic. | `CustomEvent<{ tableId: string; columnKey: string; sortingDirection: "asc" \| "desc"; }>` |
+| `tdsSort` | Sends unique Table identifier, column key and sorting direction to the tds-table-body component, can also be listened to in order to implement custom-sorting logic. | `CustomEvent<{ tableId: string; columnKey: string; sortingDirection: "desc" \| "asc"; }>` |
```

Comment:
Would be great to solve this linter issue.

Reply:
It is when we build, So should be fixed with a commit? isnt it building this readme correct?

Reply:
Yes, but I don't think we have specified the "correct" order in our lint config. So if someone has other settings locally it gonna be overridden